docs/docs/configuration/topic-preview.md

```diff
@@ -0,0 +1,9 @@
+# Topic preview
+
+See [Topic preview](/user/topic-preview) section for description of this feature.
+
+Option                                      | Description                                        | Default value
+------------------------------------------- | -------------------------------------------------- | -------------
+frontend.message.preview.enabled            | enable message preview                             | false
+frontend.message.preview.size               | maximum number of messages per topic               | 3
+frontend.message.preview.log.persist.period | how often to refresh previewed messages in seconds | 30
```

Comment:
We could add "seconds" suffix for ease of understanding without checking the docs.